charts/nfdiv-frontend/values.aat.template.yaml

```diff
@@ -8,6 +8,7 @@ nodejs:
     primaryIngressHost: ${SERVICE_FQDN}
   environment:
     OAUTH_CLIENT_REDIRECT: https://${SERVICE_FQDN}/oauth2/callback
+    IDAM_TOKEN_CACHE: true
```

Comment:
do you need to set this, since you're doing it in flux?

Reply:
@nitinprabhu said this affects staging env, which is distinct from AAT (which is configured using flux)